src/deltacode/__init__.py

```diff
@@ -283,9 +283,17 @@ class Delta(object):
     def __init__(self, score=0, new_file=None, old_file=None):
         self.new_file = new_file if new_file else None
         self.old_file = old_file if old_file else None
+        self.has_license = False
+        self.has_copyright = False
```

Comment:
Why do we need these again?

Reply:
@majurg I provided it as attributes of every delta objects,as we can just check these booleans for determining weather the respective deltas has a license or copyright(either new or old file)
Depending on these booleans we can actually use the license Expression  Library